src/modules/song/api.ts

```diff
@@ -324,6 +334,27 @@ export const extendedApi = api.injectEndpoints({
         }
       },
     }),
+    replyToCollaboration: build.mutation<void, ReplyCollaborationRequest>({
```

Comment:
I try to keep the endpoints as close as possible to the actual name on the server so that we can reference them easier. I dont think the invite helps much imo